app/models/solr_document.rb

```diff
@@ -117,6 +117,19 @@ def to_semantic_values
   attribute :document_formats, :array, FORMAT_KEY
   attribute :live_lookup_id, :string, 'uuid_ssi'
 
+  def db_az_subject
+    self[:db_az_subject] if is_a_database?
+  end
+
+  def finding_aid
+    index_links&.finding_aid if has_finding_aid?
+  end
+
+  def summary_data
+    @summary_data ||= SummaryDataComponent.new(document: self)
```

Comment:
I don't think we should initialize components in the model layer.  I think this might be a good one to move to a special case in `ResultsMetadataComponent`